src/XMakeTasks/Exec.cs

```diff
@@ -124,6 +128,14 @@ protected override Encoding StandardErrorEncoding
         }
 
         /// <summary>
+        /// Whether or not to use UTF encoding for the cmd file and console window.
+        /// Values: Always, Never, Detect
+        /// If set to Detect, UTF8 will only be used when the Command / WorkingDirectory
+        /// cannot be encoded with the current code page.
```

Comment:
Something about this reads awkward to me. Maybe "If set to Detect, the current code page will be used unless it cannot represent the string. In that case, UTF-8 is used."?